server/src/main/java/io/littlehorse/common/Storeable.java

```diff
@@ -44,10 +44,12 @@ public static StoreableType getStoreableType(Class<? extends Storeable<?>> cls)
                 return StoreableType.TAG;
             case "ScheduledTaskModel":
                 return StoreableType.SCHEDULED_TASK;
-            case "WfMetricUpdate":
-                return StoreableType.WF_METRIC_UPDATE;
             case "TaskMetricUpdate":
```

Comment:
If we're removing WfMetricUpdate, I vote we remove TaskMetricUpdate too. I don't see why they should have different mechanisms.

Reply:
Will be addressed in my next PR.